controllers/articlesController.js

```diff
@@ -44,24 +45,49 @@ const getCommentsByArticleId = (req, res, next) => {
 
 const postCommentsByArticleId = (req, res, next) => {
   const { username, body } = req.body;
-  const article_id = req.params.article_id;
-  console.log(article_id, "<<< article_id");
+  const { article_id } = req.params;
+  // console.log(article_id, "<<< article_id");
 
   const comments = {
     username,
     body,
     article_id,
   };
-  console.log(comments, "<<< comments object");
+  // console.log(comments, "<<< comments object");
 
   fetchArticleById(article_id)
     .then(() => {
       return insertCommentsByArticleId(comments);
     })
     .then((comment) => {
-      console.log(comment, "<<< insert comments");
+      // console.log(comment, "<<< insert comments");
       res.status(201).send({ comment });
     })
+    .catch((err) => {
+      // console.error(err, "<<< error");
+      next(err);
+    });
+};
+
+const patchArticleByArticleId = (req, res, next) => {
+  const { inc_votes } = req.body;
+  const article_id = req.params.article_id;
+  console.log(article_id, "<<< article_id");
+  console.log(req.body, "<<< req.body.inc_votes");
+
+  const votes = {
+    inc_votes,
+  };
+  console.log(votes, "<<< votes object");
+
+  fetchArticleById(article_id)
```

Comment:
A check on the article (if this is what it is) is not always necessary - if you try to patch an article with an id and it is the wrong data type (banana for id) PSQL will throw an error - if it returns an empty array of rows you can infer that it doesn't exist (9999 for id)